formPage.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+?>
+
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+    <title>My Bike Collection</title>
+    <link href="https://fonts.googleapis.com/css?family=M+PLUS+1p|Permanent+Marker&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="Normalize.css">
+    <link rel="stylesheet" type="text/css" href="formStyles.css">
+</head>
+
+<body>
+<section>
+    <h1>Add a bike</h1>
+
+    <div class="addContainer">
+        <div class="buttonContainer">
+            <a href="index.php"><button type="button">BACK TO COLLECTION</button><a/>
+        </div>
+    </div>
+
+    <div class="addBikeForm">
+        <form class="formLeft" method="post" action="functions.php">
+            1. Manufacturer: <br>
+            <input type="text" name="manufacturer" required><br>
+            2. Model: <br>
+            <input type="text" name="model" required><br>
+            3. Wheelset: <br>
+            <select name="wheelset" required>
+                <option value="Mavic Aksium">Mavic Aksium</option>
+                <option value="Compagnolo Scirocco CX">Compagnolo Scirocco CX</option>
+                <option value="DT R460">DT R460</option>
+                <option value="Zipp 303">Zipp 303</option>
+                <option value="Hope 20Five RS4">Hope 20Five RS4</option>
+            </select> <br>
+            4. Groupset<br>
+            <select name="groupset" required>
+                <option value="Tiagra 50-34/11-28">Tiagra 50-34/11-28</option>
+                <option value="Shimano GRX 600 2x 46-30/11-34">Shimano GRX 600 2x 46-30/11-34</option>
+                <option value="Shimano 105 52-36/11-28">Shimano 105 52-36/11-28</option>
+                <option value="Shimano Dura-Ace Di2 52-36/11-28">Shimano Dura-Ace Di2 52-36/11-28</option>
+                <option value="Campagnolo Potenza 52-36/11-29">Campagnolo Potenza 52-36/11-29</option>
+            </select>
+        </form>
+
+        <form class="formRight" method="post" action="functions.php" required>
```

Comment:
This cant be in a separate form element as it will not be submitted with the above form, merge into one form and place divs inside for formLeft and formRight

Reply:
Sorted.